addons/spreadsheet_account/models/account.py

```diff
@@ -130,3 +158,28 @@ def get_account_group(self, account_types):
         )
         mapped = dict(data)
         return [mapped.get(account_type, []) for account_type in account_types]
+
+    @api.model
+    def get_residual_amount(self, args_list):
+        results = []
+        for args in args_list:
+            company_id = args['company_id'] or self.env.company.id
+            domain = self._build_move_lines_domain(args, error_domain=False)
+            [(amount_residual_currency_sum,)] = self.env["account.move.line"].with_company(company_id)._read_group(domain, aggregates=['amount_residual_currency:sum'])
```

Comment:
Does it make sense to have the amount in foreign currency? I would have expected `amount_residual`